### PR TITLE
LPS-162545 Set correct liferay bom version in pom.xml for maven workspace template

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
@petershin here is the test result https://github.com/simonjhy/liferay-portal/pull/77. Thanks

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -12,24 +12,12 @@
...
-		<liferay.bom.version>7.2.1-1</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.2.1-ga2/liferay-ce-portal-tomcat-7.2.1-ga2-20191111141448326.tar.gz</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.3")})
-		<liferay.bom.version>7.3.7</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.3.7-ga8/liferay-ce-portal-tomcat-7.3.7-ga8-20210610183559721.tar.gz</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.4")})
-		<liferay.bom.version>7.4.3.36</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.4.3.36-ga36/liferay-ce-portal-tomcat-7.4.3.36-ga36-20220802152002501.tar.gz</liferay.workspace.bundle.url>
-#else
+
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml:15]: Brian might ask about this change and want it to be set automatically like before.

If that's not possible or this is better, its fine to leave as is.

**Review by petershin — COMMENTED** [review]
In addition to the comments, there may be conflicts with this pull: https://github.com/petershin/liferay-portal/pull/1791

File: modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java [hunk 2]
```diff
@@ -361,6 +361,28 @@ public void testComparePortalToolsBundleSupportPluginVersions()
 				"\n\t\t\t\t<version>" + portalToolsBundleSupportVersion);
 	}
 
+	@Test
+	public void testCreateMavenWorksapce() throws Exception {
+		//		Assume.assumeTrue(isBuildProjects());
```
[the remark this answers is not in the document]
Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java:366]: done

File: modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java [hunk 3]
```diff
@@ -361,6 +361,28 @@ public void testComparePortalToolsBundleSupportPluginVersions()
...
+
+		String liferayVersion = "7.2.10.4";
+
+		File destinationDir = temporaryFolder.newFolder("mavenWorkspace");
+
+		String groupId = "com.test";
```
[the remark this answers is not in the document]
Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-workspace/src/test/java/com/liferay/project/templates/workspace/ProjectTemplatesWorkspaceTest.java:372]: done

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml:15]: we do not want to hard code to set liferay bom version. User will set exactly bom version when they use this template to init a liferay maven workspace project. When user use blade or pure maven archetype to create a maven workspace, he must to provide the exactly liferay bom version and we do not need to use hard code bom version any more.

**Comment by simonjhy** [comment]
OK I will fix the conflict

**Comment by simonjhy** [comment]
@petershin I rebased and I did not find conflict notification.

**Pull request closed without merging**